app/wasm/test/custom_msg_test.go

```diff
@@ -209,6 +209,163 @@ func TestSwapMsg(t *testing.T) {
 				sdk.NewInt64Coin("uosmo", 2000000),
 			},
 		},
+
+		"exact in: 2 step multi-hop": {
+			msg: func(state BaseState) *wasmbindings.SwapMsg {
+				return &wasmbindings.SwapMsg{
+					First: wasmbindings.Swap{
+						PoolId:   state.StarPool,
+						DenomIn:  "ustar",
+						DenomOut: "uosmo",
+					},
+					// Note: you must use empty array, not nil, for valid Rust JSON
```

Comment:
Note is now outdated / irrelevant?

Reply:
Good point, there are 4 places this note was no longer needed (cut and pasted).
Removing them